File: CCC/2022Senior/Q2/q2.py
n = int(input())
sameGroup = []
differentGroup = []
currentGroup = []
for i in range(n):
    sameGroup.append(input().split(" "))
n = int(input())
for i in range(n):
    differentGroup.append(input().split(" "))
n = int(input())
for i in range(n):
    currentGroup.append(input().split(" "))
count = 0


for same in sameGroup:
    for current in currentGroup:
        if same[0] in current:
            if same[1] not in current:
                count += 1
                status = False
                break
for different in differentGroup:
    for current in currentGroup:
        if different[0] in current:
            if different[1] in current:
                count += 1
                status = False
                break

print(count)

File: CCC/2022Senior/Q2/q2.java
package Q2;

import java.util.Scanner;

public class q2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n;

        n = scanner.nextInt();
        scanner.nextLine();
        String[][] sameGroup = new String[n][2];
        for (int x = 0; x < n; x++) {
            sameGroup[x] = scanner.nextLine().split(" ");
        }
        n = scanner.nextInt();
        scanner.nextLine();
        String[][] differentGroup = new String[n][2];
        for (int x = 0; x < n; x++) {
            differentGroup[x] = scanner.nextLine().split(" ");
        }
        n = scanner.nextInt();
        scanner.nextLine();
        String[][] currentGroup = new String[n][3];
        for (int x = 0; x < n; x++) {
            currentGroup[x] = scanner.nextLine().split(" ");
        }

        for (String same[] : sameGroup) {
            for (String current[] : currentGroup) {
                for (String c : current) {
                    if (same[0].equals(c)) {
                        break;
                    }
                    System.out.println("Same");
                }
            }

        }
    }
}